app/admin/members.rb

```diff
@@ -43,7 +43,7 @@
     column :phone_number
     column :role
     column(:group) { |member| member.groups.map(&:name).join(', ') }
-    column :worked_hours_of_three_last_months_csv_version
+    column(t('active_admin.resource.index.worked_hours')) { |member| MemberDecorator.new(member).hours_worked_in_the_three_last_months(csv: true) }
```

Comment:
I think it should also work with just `t '.worked_hours'`. Does it?

Reply:
don't work wtih csv